### DISCUSSION TITLE
nxt file root and path

### DISCUSSION THREAD

**Discussion by Abdelrahimtd** [nxt-dev/nxt_editor#170] [Q&A]
how  i can set env for open rig project correctly ? 
i get this error when open biped_base_rig.txt and Run it !
please just make a demo video for it , how we can use open rig files in maya 
https://drive.google.com/file/d/1EgU8XELPLMx6W2m2tEs2kBjLJqSjHHQ4/view?usp=sharing

**Comment by ZachGray** [comment]
This looks like an import error of the supporting modules for open rig.

The example in the openRig readme suggest how to add that to your python path.

```
import os, sys

# repo path is the directory your repo is located. Where you cloned you're repo. 
repo_path = 'C:/Users/{user}/Desktop/squarebit'
openrig_path = os.path.join(repo_path, 'openrig')
nxt_roots = '{anotherPathToAssets}/character;{}/openrig/archetypes'.format(openrig_path)

sys.path.append(openrig_path)
sys.path.append(repo_path)
```

The issue is with your repo_path or openrig_path, as the file does exist. 
https://github.com/squarebit-studios/openrig/blob/main/openrig/maya/riglib/spline.py 

You may need to specify the directory _above_ the module that you are trying to import.
https://chrisyeh96.github.io/2017/08/08/definitive-guide-python-imports.html

Open rig is provided by squarebit. I think they are available for consulting if you'd like to learn how to implement something like that at your studio. The project isn't really plug and play and user facing like mGear. It assumes familiarity with python.
